### PR TITLE
kafka-exporter-scratch and ubi creation

### PR DESCRIPTION
**Pull request by Mak0tin**
I've updated some go versions (client golang, crypto, net, sys, text) as they had vulnerabilities

### CODE REVIEW

**Comment by carlosadiegosysdig** [comment]
@Mak0tin  Please add the Github Actions

File: .dockerignore [hunk 1]
```diff
@@ -1,4 +1,3 @@
 vendor
 Dockerfile
-Makefile
```
Comment (carlosadiegosysdig) [inline .dockerignore:3]: Did you make any changes to the Makefile, that led you to see that the gitignore was ignoring its changes? 
If you made any changes to the Makefile, they have not been commited. But maybe you didn't change it at all. Just wanted to confirm it.

**Comment by Mak0tin** [comment]
> @Mak0tin Please add the Github Actions

Added to PR

**Comment by carlosadiegosysdig** [comment]
I think the PR looks good! We need to wait for the Quay public repo to be created

**Review by carlosadiegosysdig — APPROVED** [review]

**Pull request merged**